recipes/tinyalsa/all/conanfile.py

```diff
@@ -0,0 +1,56 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class TinyAlsaConan(ConanFile):
+    name = "tinyalsa"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/tinyalsa/tinyalsa"
+    topics = ("conan", "tiny", "alsa", "sound", "audio", "tinyalsa")
+    description = "A small library to interface with ALSA in the Linux kernel"
+    options = {"shared": [True, False]}
+    default_options = {'shared': False}
```

Comment:
Any reason why fPIC option is missing?

Reply:
fPIC is not there because it is hardcoded in the makefile. I'm assuming it is needed since the dev explicitly put it there

Reply:
Current CMakeLists in master branch doesn't force PIC. I guess patching Makefile to remove -fPIC should be fine.

Reply:
Oohh, forgot that `all` target builds both static and shared, what a mess.

Reply:
I added a option to build with/without utils. Since it cant be controlled from the command line I just remove the `bin` folder if the consumer does not want the utils.

Reply:
maybe a comment in recipe it would be nice. this recipe is a rare case and in the future, for any update, people will need to check again this PR.